test/micro/org/openjdk/bench/java/lang/foreign/AllocTest.java

```diff
@@ -84,18 +62,51 @@ public long alloc_unsafe_arena() {
         }
     }
 
-    public static class CallocArena implements Arena {
```

Comment:
The following changes seem to be here to support 32-bit platforms? Could you please do that in a separate PR?